test/WebJobs.Script.Tests.Integration/TestScripts/CSharp/Scenarios/run.csx

```diff
@@ -36,6 +36,21 @@ public static void Run(ScenarioInput input, out string blob, TraceWriter trace,
             ["MyCustomMetricProperty"] = 100
         });
     }
+    else if (input.Scenario == "beginScope")
+    {
+        using(logger.BeginScope("{customKey1}", "customValue1"))
```

Comment:
Instead of creating this as an end to end test, can you make this a unit test? Script runs are hitting ~20min, I'd love to have more focused testing